acceptance/tests/config/apply_file_metadata_specified_in_config.rb

```diff
@@ -23,4 +25,9 @@
   on(agent, "stat --format '%U:%G %a' #{logdir}") do
     assert_match(/root:root 700/, stdout)
   end
+
+  teardown do
+    on(agent, puppet('config', 'set', 'logdir', "'#{logdir} { owner = root, group = root, mode = 0700 }'", '--confdir', get_test_file_path(agent, '')))
+    on(agent, puppet("apply -e \"#{logdir_orig}\""))
+  end
```

Comment:
@johnduarte I'm not following why these changes are necessary. Previously, the test created a temp confdir, and pointed `logdir` to another tempdir:

```
logdir = get_test_file_path(agent, 'log')
...
on(agent, puppet('config', 'set', 'logdir', "'#{logdir} { owner = root, group = root, mode = 0700 }'", '--confdir', get_test_file_path(agent, '')))
```

Then it executed `puppet apply` with the temp confdir:

```
on(agent, puppet('apply', get_test_file_path(agent, 'site.pp'), '--confdir', get_test_file_path(agent, '')))
```

So I would expect `puppet` to use the temp `logdir` specified in the temp `confdir`